unit_test.py

```diff
@@ -0,0 +1,1371 @@
+# This is file contains a unit test suit which completely test the backend
```

Comment:
Also you should put this in a directory called test, along with your db popluation files.